common/validations.js

```diff
@@ -193,3 +193,9 @@ export const isUnityFile = async (file) => {
     return false;
   }
 };
+
+export const isUsernameMatch = (data) => {
+  if(data.input == data.username) {
```

Comment:
Use `===`

Also this function would be unusable without passing in an object, why not just make it a proper string compare function and do `export const isUsernameMatch = (source, candidate)`